toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -334,6 +340,46 @@ Must be a multiple of 1 MiB.
 
 The partitions to provision on the disk.
 
+## pxe type
+
+Specifies the PXE-specific configuration for the generated OS artifacts.
+
+<div id="pxe-isoimageurl"></div>
+
+### isoImageUrl [string]
+
+Specifies the URL of the ISO image to download at boot time.
+The ISO image must be a LiveOS ISO image generated by the Azure Linux Image
+Customizer. The booting process will pivot to the root file system embedded
+in the ISO image after downloading it.
+
+For example:
+- ```yaml
+  pxe:
+    isoImageUrl: http://ip/iso-publish-path/my-liveos.iso
```

Comment:
That doesn't have to an an IP. hostnames would work too AFAIK.
Also `ftp`, `nfs`, `sftp` are all supported and doesn't have to be `http`

Reply:
Updated to indicate hostname and other protocols.
Note that the original PR only supported http though - since it relied on our custom downloader (not dracut's livenet downloader).